try.html

```diff
@@ -538,6 +538,10 @@ <h3 id="version"> Version </h3>
   <td><img src='/cpan/v/Config-Augeas.svg' alt=''/></td>
   <td><code>https://img.shields.io/cpan/v/Config-Augeas.svg</code></td>
   </tr>
+  <tr><th data-keywords='R'> CRAN: </th>
+  <td><img src='/cran/v/devtools.svg' alt=''/></td>
+  <td><code>https://img.shields.io/cran/v/devtools.svg</code></td>
+  </tr>
```

Comment:
I'll add the changes but I can't verify them beyond the CI tests as I don't have a matching dev-system here anymore.